### DISCUSSION TITLE
Block are build with tailwind but not using VuePrime component?

### DISCUSSION THREAD

**Discussion by sasafister** [primefaces/community#2934] [PrimeVue Blocks]
I assumed that blocks are build with VuePrime components, but they are just tailwind templates with some of the elements of VuePrime like:

I wasn't expecting that :(

```
import IconField from 'primevue/iconfield';
import InputIcon from 'primevue/inputicon';
import InputText from 'primevue/inputtext';
import OverlayBadge from 'primevue/overlaybadge';
```

**Comment by lukas-dne** [comment]
My team was also very disapointed to discover this issue! The same problem appears in the Figma files for PrimeBlocks and PrimeVue. @primetek It  should be made clear to the customers upfront buying, that you have to manually map all of the elements (like buttons, inputs, ect) used in PrimeBlocks to actual PrimeVue compontents. **And you have to do it in two times:** in Figma and in the code. That is really a pain in the ass :(

**Comment by dianedef** [comment]
wow thanks for sparing me this bad buy
